scripts/adminrouter-agent-setup.ps1

```diff
@@ -113,6 +113,12 @@ function Add-Routes{
     Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST'  -filter "system.webServer/rewrite/rules/rule[@name='Reverse Proxy for the DC/OS Diagnostics agent service']/action" -name "type" -value "Rewrite"
     Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST'  -filter "system.webServer/rewrite/rules/rule[@name='Reverse Proxy for the DC/OS Diagnostics agent service']/action" -name "url" -value "http://localhost:9003/{R:0}"
 
+    Write-Output "setup url rewrite rule for DC/OS Metrics service"
+    Add-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST'  -filter "system.webServer/rewrite/rules" -name "." -value @{name='Reverse Proxy for the DC/OS Metrics agent service';stopProcessing='True'}
+    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST'  -filter "system.webServer/rewrite/rules/rule[@name='Reverse Proxy for the DC/OS Metrics agent service']/match" -name "url" -value "^system/v1/metrics/(.*)"
+    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST'  -filter "system.webServer/rewrite/rules/rule[@name='Reverse Proxy for the DC/OS Metrics agent service']/action" -name "type" -value "Rewrite"
+    Set-WebConfigurationProperty -pspath 'MACHINE/WEBROOT/APPHOST'  -filter "system.webServer/rewrite/rules/rule[@name='Reverse Proxy for the DC/OS Metrics agent service']/action" -name "url" -value "http://localhost:9000/{R:1}"
```

Comment:
Can you use `$METRICS_AGENT_PORT` variable instead of the hard-coded `9000` value ? You already source `variables.ps1` here and it is accessible.

Reply:
I will do that for $DIAGNOSTICS_AGENT_PORT and $METRICS_AGENT_PORT